src/v2/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -239,8 +247,12 @@ export class OverviewRoute extends React.Component<OverviewRouteProps, {}> {
         </Media>
 
         <Media at="xs">
+          <ArtistTopWorksRail artist={artist} />
```

Comment:
Should you pass the `onOverviewTab` prop here too? It seems like it would still apply.